priv/repo/migrations/20201004171940_add_normalized_name.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Ferry.Repo.Migrations.AddNormalizedName do
+  use Ecto.Migration
+
+  def change do
+    alter table("groups") do
+      add :normalized_name, :text, null: false
```

Comment:
Also, I'm considering using `slug` instead of `normalized_name` as the name for the field, just because it reads easier, and same length as `name`. Let me know if you are happy with that?

Reply:
Also thought of that ;)

Reply:
Yes definitely slug fits a lot better

Reply:
I think there is a `slug` column in table `groups` already, btw. Also, might be worth adding the same to the `items` and `mods` tables as well.

Reply:
Thanks. Will do.

Reply:
also just realised, table `aid__item_categories` might not even be used atm.. Probably we could even delete it from the schema, or we can do it in a separate PR.

Reply:
I have made the necessary updates. Please confirm that I haven't missed anything.